pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -298,6 +298,7 @@ protected void internalRevokePermissionsOnTopic(AsyncResponse asyncResponse, Str
         }
 
         validateAccessForTenantCf
+                .thenCompose(__ -> validatePoliciesReadOnlyAccessAsync())
```

Comment:
In order to keep the same logic, better to move this after `validateAdminAccessForTenantAsync`